File: puzzles/hr/MakeAnagram.cpp
#include <iostream>
#include <set>
#include <string>

using namespace std;

/*
 * Complete the 'makeAnagram' function below.
 *
 * The function is expected to return an INTEGER.
 * The function accepts following parameters:
 *  1. STRING a
 *  2. STRING b
 */

int makeAnagram(string a, string b) {
  set<char> f(a.begin(), a.end());
  set<char> s(b.begin(), b.end());
  set<char> u = f & s;
}

int main() {
  ofstream fout(getenv("OUTPUT_PATH"));

  string a;
  getline(cin, a);

  string b;
  getline(cin, b);

  int res = makeAnagram(a, b);

  fout << res << "\n";

  fout.close();

  return 0;
}

File: puzzles/hr/MakeAnagram.py
#!/bin/python3
import os
# Complete the 'makeAnagram' function below.
#
# The function is expected to return an INTEGER.
# The function accepts following parameters:
#  1. STRING a
#  2. STRING b
def makeAnagram(a, b):
    # Write your code here
    set1 = set(a)
    set2 = set(b)
    setUnion=set1|set2
    result1 = setUnion - set1
    result2 = setUnion - set2
    return len(result1)+len(result2)
if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')

    a = input()

    b = input()

    res = makeAnagram(a, b)

    fptr.write(str(res) + '\n')
    fptr.close()
